hello-signing.md

```diff
@@ -0,0 +1,145 @@
+- Import public content to a private registry
+- Promote from dev through production
+
+![](./media/notary-e2e-scenarios.svg)
+
+To illustrate these scenarios we will introduce two example companies:
+
+- **Wabbit Networks**: A small software company that produces network monitoring software
+- **ACME Rockets**: A consumer, who runs network monitoring software
+
+ACME Rockets doesn't know about Wabbit Networks, but finds their [net-monitor software in Docker Hub](https://hub.docker.com/r/wabbitnetworks/net-monitor).
+Since they've never heard of Wabbit Networks, they're a little reluctant to run network software without some validations.
+ACME Rockets has a policy to only import Docker Hub certified content, and other approved vendors.
+
+Wabbit Networks works with Docker Hub to get certified, to help with their customer confidence.
+
+ACME Rockets will only deploy software that's been scanned and approved by the ACME Rockets security team. They know it's been approved because all approved software has been signed by the ACME Rockets security team.
+
+## Getting Started
+- Setup a few environment variables.  
+  >Note see [Simulating a Registry DNS Name](#simulating-a-registry-dns-name) to use `registry.wabbit-networks.io`
+  ```bash
+  export PORT=5000
+  export REGISTRY=localhost:${PORT}
+  export REPO=${REGISTRY}/net-monitor
+  export IMAGE=${REPO}:v1
+  ```
+- Install [Docker Desktop](https://www.docker.com/products/docker-desktop) for local docker operations
+- Run a local instance of the [CNCF Distribution Registry][cncf-distribution]
+  ```bash
+  docker run -d -p ${PORT}:5000 ghcr.io/oras-project/registry:latest
+  ```
+- Acquire the Notation CLI  
+Notation releases can be found at: [Notation Releases][notation-releases]  
+  ```bash
+  #LINUX, including WSL
+  curl -Lo notation.tar.gz https://github.com/shizhMSFT/notation/releases/download/v0.5.2/notation_0.5.2_linux_amd64.tar.gz
+  tar xvzf notation.tar.gz -C ~/bin notation
+  ```
+
+## Building and Pushing
+- Build and Push the `net-monitor` software
+  ```bash
+  docker build -t $IMAGE https://github.com/wabbit-networks/net-monitor.git#main
+
+  docker push $IMAGE
+  ```
+- List the image, and any associated signatures
+  ```bash
+  notation list $IMAGE
+  ```
+  At this point, the results are empty, as there are no existing signatures
+
+## Signing a Container Image
+
+To get things started quickly, the Notation cli supports generating self signed certificates. As you automate the signing of content, you will most likely want to create and store the private keys in a key vault. (Detailed production steps will be covered later)
+
+- Generate a self-signed test certificate for signing artifacts
+  The following will generate a self-signed X.509 certificate under the `~/config/notation/` directory
+  ```bash
+  notation cert generate-test --default "wabbit-networks.io"
+  ```
+- Sign the container image
+  ```bash
+  notation sign $IMAGE
```

Comment:
I'd prefer being explicit about the key being used and not using a default, we can address that later.

Reply:
We did want to enable the simple/default scenarios but appreciate the concern. 
@gokarnm, can you open an issue for this?

Reply:
Opened #91